runtime/convertValues.go

```diff
@@ -457,6 +469,43 @@ func importPathValue(v cadence.Path) interpreter.PathValue {
 	}
 }
 
+func importTypeValue(
+	_ *interpreter.Interpreter,
+	v cadence.Type,
+) (
+	interpreter.TypeValue,
+	error,
+) {
+	return interpreter.TypeValue{
+		Type: ImportType(v),
+	}, nil
+}
+
+func importCapability(
+	_ *interpreter.Interpreter,
+	path cadence.Path,
+	address cadence.Address,
+	borrowType cadence.Type,
+) (
+	interpreter.CapabilityValue,
+	error,
+) {
+
+	switch borrowType.(type) {
+	case cadence.ReferenceType:
+		return interpreter.CapabilityValue{
```

Comment:
Careful here: It is only safe to import **public** capabilities, i.e. ones with a public path. See the description of #712. 

The tests below only check that the import of a capability with a storage path succeeds, which should not. Could you please also add tests for the other path domains and assert that only the import of public ones succeeds?

I'm not quote sure if this should be checked here. 
@SupunS Where would the proper place be? I remember you had worked on checking imported values before, but I don't quite remember the mechanics.

Reply:
I think that validation should probably go to `CapabilityDynamicType.IsImportable()`. Checking `isImportable` is already done when importing values, so it should get automatically validated there. We would have to include pathType info to the `CapabilityDynamicType` to make it work.